frontend/ee/components/UsersPage/UsersTable.jsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { Pagination } from '@/_components';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+const UsersTable = ({
+  isLoading,
+  users,
+  archivingUser,
+  unarchivingUser,
+  meta,
+  generateInvitationURL,
+  invitationLinkCopyHandler,
+  unarchiveOrgUser,
+  archiveOrgUser,
+  pageChanged,
+  darkMode,
+}) => {
+  const tableRef = React.createRef(null);
+
+  function calculateOffset() {
+    const elementHeight = tableRef.current.getBoundingClientRect().top;
+    return window.innerHeight - elementHeight;
+  }
+
+  const setWidth = () =>
+    (document.querySelector('.users-pagination').style.width = `${tableRef.current.offsetWidth}px`);
+
+  React.useEffect(() => {
+    window.addEventListener('resize', setWidth);
+    return () => window.removeEventListener('resize', setWidth);
+  });
+
+  React.useEffect(() => {
+    setWidth();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [tableRef]);
+
+  return (
+    <div className="container-xl">
+      <div className="card">
+        <div
+          className="card-table fixedHeader table-responsive table-bordered"
+          ref={tableRef}
+          style={{
+            maxHeight: tableRef.current && calculateOffset(),
+          }}
+        >
+          <table data-testid="usersTable" className="table table-vcenter" disabled={true}>
+            <thead>
+              <tr>
+                <th data-cy="name-title">Name</th>
+                <th data-cy="email-title">Email</th>
+                {users && users[0]?.status && <th data-cy="status-title">Status</th>}
+                <th className="w-1"></th>
+              </tr>
+            </thead>
+            {isLoading ? (
+              <tbody className="w-100" style={{ minHeight: '300px' }}>
+                {Array.from(Array(4)).map((_item, index) => (
```

Comment:
Probably can use the package `react-loading-skeleton` which is installed in the fe repo already